.travis.yml

```diff
@@ -26,6 +26,56 @@ stages:
 matrix:
   fast_finish: true
   include:
+    -
```

Comment:
This PR is great! The only thing I think we should change is that we should either unmanage the `.travis.yml` from the `.sync.yml` or add these includes to it. Otherwise, this will get blown away on the next `pdk` update.

Reply:
I removed it from management, i deliberated adding it , but if we are going to start managing and unmanaging OS's maybe we should leave it out of management until the contents stablisise